cluster/config-defaults.yaml

```diff
@@ -132,6 +132,16 @@ skipper_redis_pool_timeout: "250ms"
 skipper_redis_read_timeout: "25ms"
 skipper_redis_write_timeout: "25ms"
 
+# "true" enable autoscaling
+# "false" remove autoscaling
+# "" do not change
+skipper_ingress_redis_autoscaling: ""
+skipper_ingress_redis_swarm_enabled: "true"
+skipper_ingress_redis_target_average_utilization_cpu: "30"
```

Comment:
We currently have a paging alert triggering on 85% usage. So we can probably comfortably set this higher now that it will also be automatic and not depend on a human waking up and scaling :)

Reply:
I know but this is average and I want to try to scale up as fast as we need and not only when it's too late. 
Let's keep it like this and learn by inspecting metrics and set better target if we see that it's a waste of money.